azurerm/internal/services/applicationinsights/resource_arm_application_insights.go

```diff
@@ -89,6 +89,19 @@ func resourceArmApplicationInsights() *schema.Resource {
 
 			"tags": tags.Schema(),
 
+			"daily_cap": {
+				Type:         schema.TypeFloat,
+				Optional:     true,
+				Default:      100,
```

Comment:
Could we make this as computed and get the default from the api?

Reply:
Removed default switched to computed.